### ISSUE TITLE
[Bug] Low Quality Preview Images / Game Screen Shots & Cropped Images

### ISSUE THREAD

**Issue by Xykrotik** [rommapp/romm#1229]
**RomM version**
3.5.1

**Describe the bug**
Each game/rom has preview images / screen shots from the game below the details panel on each game/rom's page. These are compressed/low quality, with no way of changing it that I can find. (Frankly, unsure if it's a RomM issue or something on my end, unsure where to check for this) They are also cropped strangely compared to the same image on the associated database site. (In my case, I'm using IGDB)

**To Reproduce**

1. Go to game/rom of your choice.
2. Click on game/rom to be taken to it's associated metadata page.
3. Scroll down to the bottom, below the game's details.
4. See preview images & screenshots pulled from remote info database

**Expected behavior**
Uncompressed images, uncropped images. (Perhaps ability to add custom screenshots of games ourselves)

**Screenshots**
![image](https://github.com/user-attachments/assets/c45e1a11-e704-450c-9537-64f22e0a4b12)
![image](https://github.com/user-attachments/assets/6aeb1cb4-578a-46ad-b84e-1e30b2cd8922)
![image](https://github.com/user-attachments/assets/a8f5f0a7-5d30-422f-b4d7-29dd38064959)

**Desktop (please complete the following information):**

- OS: Accessing RomM from a Windows 10 PC, Hosting RomM on Debian 12 Install
- Browser: Vivaldi (Chromium)
- Version: 6.9.3447.51 (Stable channel) (64-bit)

**Comment by adamantike** [comment]
@Xykrotik, you can check in IGDB, to see if the pixelated images are also like that there. I found that was the case for the ones I was able to test.

Regarding the incorrect cropping, the fix was implemented in #1263 and will be included in the next release!

**Comment by Xykrotik** [comment]
@adamantike Sorry to necro this, I had thought my container/romm had been auto updating all this time, but it hadn't. Manually pulled today to the latest version but still notice the screenshots on games are cropped. I tried refreshing the metadata and that didn't help. 

Is the fix you implemented something that needs to be set/adjusted via the config file or something similar? Perhaps there's another way to refresh the screenshots pulled from IGDB than what I'm doing/ 
(I've also tried unmatching and then manually matching the game again) 

Thanks for your help!
